roles/backend_setup/README.md

```diff
@@ -186,6 +186,55 @@ Configure the ports and services related to GlusterFS, create logical volumes an
 
 ```
 
+Tests
+-----
+This role can now be tested with molecule.
+
+### Dependencies
+This setup is currently verified with the following setup
+* Centos 7
+* Docker CE
+* Python 2
+
+As we verify our tests with more distributions, we will document them.
+
+### Setup
+* Start and enable the Docker service with the following:
+
+        $ systemctl start docker
+        $ systemctl enable docker
+
+* Ensure that your user is in the docker group.
+
+        $ sudo groupadd docker
+        $ sudo usermod -aG docker $USER
+
+* Verify everything works with the following:
+
+        $ docker run hello-world
+
+* If you have SELinux enabled, run the following
+
+        $ sudo yum install libselinux-python
+
+* Create a virtualenv
+
+        $ virtualenv --system-site-packaes env
```

Comment:
minor nit --system-site-packages

Also add a step to activate the env (prior to pip install step)
$ source env/bin/activate